packages/earlybird/earlybird.1.3.0/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis: "OCaml debug adapter"
+maintainer: ["Simmo Saan <simmo.saan@gmail.com>"]
+authors: ["hackwaly@qq.com"]
+license: "MIT"
+homepage: "https://github.com/hackwaly/ocamlearlybird"
+bug-reports: "https://github.com/hackwaly/ocamlearlybird/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.12.0"}
+  "ppx_deriving" {>= "5.1"}
+  "ppx_deriving_yojson" {>= "3.6.1"}
+  "menhir" {>= "20201216" & build}
+  "menhirLib" {>= "20201216"}
+  "ocaml-compiler-libs" {>= "v0.12.3"}
+  "ppx_optcomp" {>= "v0.11"}
+  "iter" {>= "1.2.1"}
+  "lwt" {>= "5.4.0"}
+  "lwt_ppx" {>= "2.0.1"}
+  "lwt_react" {>= "1.1.3"}
+  "cmdliner" {>= "1.1.0"}
+  "logs" {>= "0.7.0"}
+  "fmt" {>= "0.8.9"}
+  "path_glob" {>= "0.2"}
+  "sexplib" {>= "v0.14.0"}
+  "csexp" {>= "1.3.2"}
+  "lru" {>= "0.3.0"}
+  "dap" {>= "1.0.6"}
+  "odoc" {with-doc}
+]
```

Comment:
```suggestion
]
conflicts: [
  "ocaml-option-bytecode-only"
]
```

I think the failure has to do with the fact that there is no native compiler for those architectures